src/System.Private.CoreLib/src/System/Environment.CoreRT.cs

```diff
@@ -53,7 +53,6 @@ internal static void FailFast(string message, Exception exception, string errorS
 
         internal static void ShutdownCore()
         {
-            // TODO: shut down threading etc.
```

Comment:
We can't do it here. As per [msdn Environment.Exit(Int32)](https://docs.microsoft.com/en-us/dotnet/api/system.environment.exit?view=netcore-3.1)
> Exit terminates an application immediately, even if other threads are running. If the return statement is called in the application entry point, it causes an application to terminate only after all foreground threads have terminated.